save-core/src/commonNonJsMain/kotlin/org/cqfn/save/core/utils/TomlUtils.kt

```diff
@@ -14,52 +14,47 @@ import org.cqfn.save.plugins.fix.FixPluginConfig
 import org.cqfn.save.plugins.fixandwarn.FixAndWarnPluginConfig
 
 import com.akuleshov7.ktoml.KtomlConf
-import com.akuleshov7.ktoml.deserializeTomlFile
 import com.akuleshov7.ktoml.exceptions.KtomlException
+import com.akuleshov7.ktoml.file.TomlFileReader
 import com.akuleshov7.ktoml.parsers.TomlParser
-import com.akuleshov7.ktoml.parsers.node.TomlNode
 import com.akuleshov7.ktoml.parsers.node.TomlTable
 import okio.Path
 
 import kotlinx.serialization.ExperimentalSerializationApi
-
-/**
- * @return all top level table nodes
- */
-fun TomlNode.getTopLevelTomlTables() = this.getRealTomlTables().filter { it.level == 0 }
+import kotlinx.serialization.serializer
+import okio.FileSystem
+import org.cqfn.save.core.files.readLines
 
 private fun Path.testConfigFactory(table: TomlTable) =
-        when (table.fullTableName.uppercase()) {
-            TestConfigSections.FIX.name -> this.createPluginConfig<FixPluginConfig>(
-                table.fullTableName
-            )
-            TestConfigSections.`FIX AND WARN`.name -> this.createPluginConfig<FixAndWarnPluginConfig>(
-                table.fullTableName
-            )
-            TestConfigSections.WARN.name -> this.createPluginConfig<WarnPluginConfig>(
-                table.fullTableName
-            )
-            TestConfigSections.GENERAL.name -> this.createPluginConfig<GeneralConfig>(
-                table.fullTableName
-            )
-            else -> throw PluginException(
-                "Received unknown plugin section name in the input: [${table.fullTableName}]." +
-                        " Please check your <$this> config"
-            )
-        }
+    when (table.fullTableName.uppercase().replace("\"", "")) {
```

Comment:
This means, that deserialized strings still have surrounding quotes?

Reply:
No, this is only regarding the table name. FullTableName is actually an internal field that should not be used in classic deserialization scenario. 

`a = "abc"` - for key-value pairs of course I do not store quotes.

But we in save want to use these names of tables, so we need to use some technical fields.